.github/workflows/continuous_integration.yml

```diff
@@ -28,8 +28,15 @@ jobs:
           git clone --depth 1 https://github.com/junegunn/fzf.git ~/.fzf
           ~/.fzf/install
 
+      - name: Install fd
```

Comment:
Had to add installation of fd because one of the new tests had to make full of it

Reply:
Actually you can use Homebrew on both macOS and Linux in GitHub Actions, so these can be simplified into `brew install fzf fd`